interface/billing/sl_receipts_report.php

```diff
@@ -7,6 +7,7 @@
  * but I wanted to make the code available to the project because
```

Comment:
Line 55: use xlt()
Line 179: remove the e parameter
Line 188: use attr()
Line 228: use xla()
Line 236: use attr()
Line 391-394: use binding in the query
Line 494-497: use binding in the query
What about line 409??? Doesn't that need to be dealt with?
Line 529-530: use binding in the query
Line 549: text() around $docname
Line 566: use binding in the query
Line 567: wrap Unknown with xl() function
Line 577: text()
Line 584: text()
Line 591-593: use binding in the query
Line 628: text() around $docname